ncm-afsclt/src/main/pan/components/afsclt/schema.pan

```diff
@@ -5,21 +5,20 @@
 
 declaration template components/afsclt/schema;
 
-include 'quattor/schema';
+include 'quattor/types/component';
 
 type component_afsclt_entry = {
    include structure_component
    
-   "thiscell"   : string         # "AFS home cell"
-   "thesecells" ? string[]       # "Cell list to authenticate to"
-   "settime"    ? boolean        # "Shall AFS client sync sys time?"
-   "cellservdb" ? string         # "Where Master CellServDB can be found"
-   "cachemount" ? string         # "AFS cache location (/usr/vice/etc/cache)"
-   "cachesize"  ? string         # "AFS cache size in kB"
-   "options"    ? nlist          # "AFS client options"
-   "enabled"    : string with match (SELF, 'yes|no') 
-   				 # "Shall AFS client be active ?"
-   "afsd_args"  ? string{} # /etc/afsd.args values for rc.afs
+   "thiscell"   : string         # AFS home cell
+   "thesecells" ? string[]       # Cell list to authenticate to
+   "settime"    ? boolean        # Shall AFS client sync sys time?
+   "cellservdb" ? string         # Where Master CellServDB can be found
+   "afs_mount"  ? string         # AFS mount point (e.g. /afs)
+   "cachemount" ? string         # AFS cache location (/usr/vice/etc/cache)
+   "cachesize"  ? string         # AFS cache size in kB
+   "enabled"    : legacy_binary_affirmation_string  # Shall AFS client be active ?
```

Comment:
IMHO this is a retrograde step for readability. I now need to locate this type defined in some other file before I can understand how to populate this mandatory field, which has no default. legacy_yes_or_no_string would be more intuitive.